src/main/java/org/example/service/RemoveSubstringService.java

```diff
@@ -0,0 +1,18 @@
+package org.example.service;
+
+public class RemoveSubstringService {
```

Comment:
Hmm....
Remove substring is a simple statement: `String.replace()`

Why you need a separate class to call this method?
Isn't it easier to simply call this method directly on the String that requires this action?

Reply:
fixed